dagshub/data_engine/model/datasource.py

```diff
@@ -120,6 +130,20 @@ def all(self) -> "QueryResult":
         self._check_preprocess()
         return self._source.client.get_datapoints(self)
 
+    def select(self, selected: List[Field]):
+        self._select = []
```

Comment:
Add a docstring explaining the purpose of the function + a usage example

Reply:
Additionally: change the signature of the function to be
```suggestion
    def select(self, *selected: Field):
        self._select = []
```
Skips you from creating a list in the arguments